target/linux/layerscape/patches-4.4/0001-arch-Introduce-post-init-read-only-memory.patch

```diff
@@ -0,0 +1,114 @@
+From 7ece66e296972744ec471d521b8ca07990e2494f Mon Sep 17 00:00:00 2001
```

Comment:
NXP LSDK will release kernel v4.9 on September 2017. We will update our targets to v4.9 in LEDE on September.
But currently customers still use v4.4, this patch-set is still needed.

May I know whether LEDE will still maintain v4.4 after v4.9 is released?